test/TrueLayer.Tests/Payouts/PayoutsApiTests.cs

```diff
@@ -45,7 +48,7 @@ public async Task CreatePayout_Returns_Empty_Response_With_Http_Status_Code_On_A
             var sut = new PayoutsApi(Mock.Of<IApiClient>(), authApiMock.Object, _trueLayerOptions);
 
             // Act
-            var actual = await sut.CreatePayout(CreatePayoutRequest(), "indepotency-key", CancellationToken.None);
+            var actual = await sut.CreatePayout(createPayoutRequest, "indepotency-key", CancellationToken.None);
 
             //Assert
             actual.ShouldNotBeNull();
```

Comment:
Should you test the other properties you added with this PR?

Reply:
Should be fixed in 8c792de.